components/raftstore/src/store/peer.rs

```diff
@@ -3447,7 +3447,7 @@ where
         send_msg.set_message(msg);
 
         if let Err(e) = trans.send(send_msg) {
-            warn!(
+            debug!(
```

Comment:
Does it mean there is no info-level log if the error is `full`?

Reply:
Yes, but there will be metrics in the error panel.

Reply:
How about adding a metric to show how many and what type of message we have dropped? We can push data to the metric wherever we have dropped a message, for example, dropping a message of a peer that we can't find in the store.

Reply:
Add comments to explain why only the full error log is removed?